woocommerce/woocommerce-ios
# Added tracking events for Downloadable Files

Closes #3381

## Description
Added the events for downloadable files.
@shiki while I was working on this issue, I discovered that we didn't add any event when the Downloadable Files are updated. Do we want to add any events there?
The feature will be enabled for everyone at the end of milestone 5. #3400

## Testing 
1. Open a product with some downloadable files.
2. Tap the cell of the Downloadable files: the event `product_detail_view_downloadable_files_tapped` should be triggered.
3. Open the downloadable files settings, and edit them: when tapping Done, the event `product_downloadable_files_settings_changed` should be triggered.
4. When you modify downloadable files (add/update/delete), the event `products_downloadable_file` should be triggered.


Update release notes:
